docs/libertyDev.md

```diff
@@ -7,6 +7,8 @@ Starting in version 3.4.1, dev mode invokes the `generateFeatures` task when the
 
 **Limitations:** This task is not supported with Spring Boot applications.
 
+This task requires applications to be installed as loose applications. Information on configuring loose applications can be found in the [deploy task parameter documentation](deploy.md#Parameters) and the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration).
```

Comment:
For me, it was not obvious that specifying a file instead of a task would mean it is not loose application. Do you think it will be obvious to an end user?

Reply:
No, I agree with you there. It's not too clear from our documentation. We don't really have anything that covers all the aspects of the loose application configuration in one spot. Would be nice to have something to point to here rather than 2 different sections of documentation that don't really give the whole picture of what is needed.